packages/playground-ui/src/ds/components/BrandLoader/brand-loader.css

```
@@ -0,0 +1,53 @@
+.brand-loader svg {
+  filter: drop-shadow(0 0 0.7px currentColor);
+  shape-rendering: geometricPrecision;
+}
+.brand-loader circle {
+  fill: currentColor;
+}
```

Stylelint (17.7.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)